move/its/sources/its.move

```diff
@@ -77,6 +78,19 @@ module its::its {
         self.address_tracker.set_trusted_address(chain_name, trusted_address);
     }
 
+    public(package) fun set_trusted_addresses(self: &mut ITS, trusted_addresses: TrustedAddresses) {
+        let (mut chain_names, mut trusted_addresses) = trusted_addresses.destroy();
+        let length = chain_names.length();
+        let mut i = 0;
+        while(i < length) {
+            self.set_trusted_address(
+                ascii::string(chain_names.pop_back()),
```

Comment:
Use string types in TrustedAddresses to avoid the separate cast?

Reply:
Because we are peeling of off bcs, we can `peel_vec_vec_u8` but we can't `peel_vec_string` which is why I chose vectors which needs the cast.

Reply:
I'm suggesting to add `peel_vec_string` in abi so it's cleaner to work with here

Reply:
But we are peeling of off bcs, not abi. Are you suggesting changing the data input entirely?

Reply:
Ah, missed that part. I'm ok with keeping it as BCS, but we should still clean this up. Let's get to it after the call contract ticket refactor